packages/suite/src/components/suite/modals/AccessCoinjoinAccount.tsx

```diff
@@ -0,0 +1,143 @@
+import { Button, P, Image, variables, H2, H3, Warning } from '@trezor/components';
+import { Modal, Translation } from '@suite-components';
+
+const SmallModal = styled(Modal)`
+    width: 560px;
+`;
+
+const Description = styled(P)`
+    font-size: ${variables.FONT_SIZE.SMALL};
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    text-align: center;
+`;
+
+const DescriptionLeft = styled(Description)`
+    text-align: left;
+`;
+
+const ImageContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 28px;
+`;
+
+const StyledImage = styled(Image)`
+    width: auto;
+    height: 150px;
+`;
+
+const StyledH2 = styled(H2)`
+    margin-top: 12px;
+    margin-bottom: 12px;
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    font-size: ${variables.FONT_SIZE.H2};
+`;
+
+const StyledH3 = styled(H3)`
+    margin-top: 12px;
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    font-size: ${variables.FONT_SIZE.H3};
+    text-align: left;
+`;
+
+const PlaceHolder = styled.div`
+    width: 100%;
+    height: 74px;
+    background-color: #ccc;
+`;
+
+type AccessCoinjoinAccountProps = Omit<
+    Extract<UserContextPayload, { type: 'access-coinjoin-account' }>,
+    'type'
+> & {
+    onCancel: () => void;
+};
+
+export const AccessCoinjoinAccount = ({ onCancel, decision }: AccessCoinjoinAccountProps) => {
+    const [step, setStep] = useState('initial');
+
+    const onContinue = () => {
+        switch (step) {
+            case 'initial':
+                return setStep('anonymity-level');
+            case 'anonymity-level':
+                // When returning `true` we successfully completed the process.
+                return decision.resolve(true);
+            // no default
+        }
+    };
+
+    const onBackClick = () => {
+        switch (step) {
+            case 'initial':
+                return decision.resolve(false);
+            case 'anonymity-level':
+                return setStep('initial');
+            // no default
+        }
+    };
+
+    const getStep = () => {
+        switch (step) {
+            case 'initial':
+                return (
+                    <>
+                        <ImageContainer>
+                            <StyledImage image="COINJOIN_MESS" />
+                        </ImageContainer>
+                        <StyledH2>
+                            <Translation id="TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_TITLE" />
+                        </StyledH2>
+                        <Description>
+                            <Translation id="TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_DESCRIPTION" />
+                        </Description>
+                        <Warning>
+                            <Translation
+                                id="TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_FEE_MESSAGE"
+                                values={{ fee: '0.03' }}
```

Comment:
where are we gonna get it from?

Reply:
@karliatto please put it in the config so we have it defined on one place only. 
probably here if you don't have any other ideas: https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/services/coinjoin/config.ts

Reply:
For a public release it will probably have to be dynamic, which doesn't change the fact that it would be good to have it in one place already.

discussion here: https://satoshilabs.slack.com/archives/CLAR68GBV/p1665482030303019?thread_ts=1665480480.161009&cid=CLAR68GBV